src/libs/ajax.ts

```diff
@@ -23,7 +23,7 @@ import { getUser } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
 
-window.ajaxOverrideUtils = {
+(window as any).ajaxOverrideUtils = {
```

Comment:
minimal conversion of ajax.js to .ts is not central to the DI patterns.  I did this just to avoid type anoyances/polution of the core code bits elsewhere.